pkg/flowaggregator/exporter/ipfix.go

```diff
@@ -234,18 +256,16 @@ func (e *IPFIXExporter) createAndSendTemplate(isRecordIPv6 bool) error {
 	} else {
 		e.templateIDv4 = templateID
 	}
-	bytesSent, err := e.sendTemplateSet(isRecordIPv6)
-	if err != nil {
+	if err := e.sendTemplateSet(isRecordIPv6); err != nil {
```

Comment:
ditto, should it flush once?

Reply:
I understand there is no point to flush here now as this is right after initialization.